ucm2/MediaTek/mtk-rt5650/HiFi.conf

```diff
@@ -0,0 +1,94 @@
+# Use case configuration for mtk-rt5650
+
+SectionVerb {
+	# ALSA PCM
+	Value {
+		TQ "HiFi"
+	}
+
+	Include.e.File "/codecs/rt5645/EnableSeq.conf"
+
+	EnableSequence [
+		cset "name='O03 I05 Switch' 1"
+		cset "name='O04 I06 Switch' 1"
+		cset "name='O09 I17 Switch' 1"
+		cset "name='O10 I18 Switch' 1"
+
+		cset "name='ADC Capture Volume' 47"
+		cset "name='IF2 ADC Mux' IF_ADC1"
+		cset "name='IN1 Boost' 0"
+		cset "name='RT5650 IF1 ADC1 Swap Mux' L/R"
+		cset "name='Stereo1 ADC1 Mux' ADC"
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker"
+
+	Value {
```

Comment:
It's just a suggestion: Move Value {} section as the last in the device definition like in other config files.